x-pack/solutions/security/plugins/security_solution/server/endpoint/lib/artifacts/lists.ts

```diff
@@ -422,29 +421,7 @@ function translateEntry(
             value: entry.value,
           };
 
-          const hasExecutableName = hasSimpleExecutableName({
-            os: os as OperatingSystem,
-            type: entry.type,
-            value: entry.value,
-          });
-
-          const existingFields = exceptionListItemEntries.map((e) => e.field);
-          const doAddPerformantEntries = !(
-            existingFields.includes('process.name') || existingFields.includes('file.name')
-          );
-
-          if (hasExecutableName && doAddPerformantEntries) {
-            // when path has a full executable name
-            // append a process.name entry based on os
-            // `exact_cased` for linux and `exact_caseless` for others
-            return appendOptimizedEntryForEndpoint({
```

Comment:
We talked offline and looks like with the better endpoint we don't need this optimization now. I'd also ask @intxgo to review this just in case.